tpr/tpr.go

```diff
@@ -107,6 +108,28 @@ func (t *TPR) Name() string { return t.name }
 // Group returns a TPR group extracted from Name. See Config.Name godoc for details.
 func (t *TPR) Group() string { return t.group }
 
+// Endpoint returns a TPR resurce endpoint registered in Kubernetes API under
+// a given namespace. Default namespace will be used when argument is an empty
+// string.
+func (t *TPR) Endpoint(namespace string) string {
+	nsResource := t.resourceName
+	if len(namespace) != 0 {
+		nsResource = "namespace/" + namespace + "/" + t.resourceName
+	}
+	return "/apis/" + t.group + "/" + t.version + "/" + nsResource
```

Comment:
I thought about path.Join already. Problem is that we'd have to create different slice depending if namespace is `""` or not.